src/kvstore/RocksEngineConfig.cpp

```diff
@@ -142,7 +148,9 @@ static const std::unordered_map<std::string, rocksdb::CompressionType> kCompress
     {"lz4hc", rocksdb::kLZ4HCCompression},
     {"zstd", rocksdb::kZSTD},
     {"zlib", rocksdb::kZlibCompression},
-    {"bzip2", rocksdb::kBZip2Compression}};
+    {"bzip2", rocksdb::kBZip2Compression},
+    {"xpress", rocksdb::kXpressCompression},
+    {"disable", rocksdb::kDisableCompressionOption}};
```

Comment:
what's the difference between disable and no?

Reply:
kNoCompression means no compression
kDisableCompressionOption is used to disable some compression options.
refer to https://github.com/facebook/rocksdb/blob/fe31dc53ca04a6241a1489be826b20218dff987e/include/rocksdb/compression_type.h#L17

Reply:
I see. Awesome job.